.github/workflows/codeql.yml

```
@@ -11,6 +11,10 @@ on:
   schedule:
   - cron: '39 10 * * 1'
 
+permissons:
```

actionlint (1.7.12)

[error] unexpected key "permissons" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)